base_rest/components/service.py

```diff
@@ -84,32 +66,28 @@ def _log_call(self, func, params, secure_params, res):
             _logger.debug(message, *args, extra=extra)
 
     def _get_schema_for_method(self, method_name):
```

Comment:
@acsonefho `get_request_schema(self, method_name)` to be consistent with the naming of schema properties